analysis_options.yaml

```diff
@@ -1,50 +1,21 @@
-# This is a copy (as of March 2021) of flutter/flutter's analysis_options file,
-# with minimal changes for this repository. The goal is to move toward using a
-# shared set of analysis options as much as possible, and eventually a shared
-# file.
-
 # Specify analysis options.
 #
-# For a list of lints, see: http://dart-lang.github.io/linter/lints/
-# See the configuration guide for more
-# https://github.com/dart-lang/sdk/tree/main/pkg/analyzer#configuring-the-analyzer
-#
-# There are other similar analysis options files in the flutter repos,
-# which should be kept in sync with this file:
-#
-#   - analysis_options.yaml (this file)
-#   - packages/flutter/lib/analysis_options_user.yaml
-#   - https://github.com/flutter/flutter/blob/master/analysis_options.yaml
-#   - https://github.com/flutter/engine/blob/main/analysis_options.yaml
-#   - https://github.com/flutter/packages/blob/main/analysis_options.yaml
-#
-# This file contains the analysis options used for code in the flutter/plugins
-# repository.
+# This file is a copy of analysis_options.yaml from flutter repo
+# as of 2022-07-27, but with some modifications marked with
+# "DIFFERENT FROM FLUTTER/FLUTTER" below. The file is expected to
+# be kept in sync with the master file from the flutter repo.
```

Comment:
Everything from here up is just comment differences from flutter/packages; aligning them help reduce noise in my diffs as I'm finishing the pre-merge alignment.